qiskit/transpiler/passes/synthesis/unitary_synthesis.py

```diff
@@ -120,6 +121,10 @@ def __init__(
             target: The optional :class:`~.Target` for the target device the pass
                 is compiling for. If specified this will supersede the values
                 set for ``basis_gates`` and ``coupling_map``.
+            fallback_on_default: specifies whether the default synthesis method in the case
+                that a non-default synthesis ``method`` is specified but is either unable
+                to synthesize the operation or the synthesized circuit does not conform to the
+                target.
```

Comment:
My english senses tell me that there is an "is used" missing here somewhere
```suggestion
            fallback_on_default: specifies whether the default synthesis method should be 
                used in case a custom synthesis ``method`` is specified, but is either unable
                to synthesize the operation, or the synthesized circuit does not conform to the
                target.
```

Reply:
your english senses are correct, fixing